cloud/google/pods.go

```diff
@@ -129,7 +127,7 @@ func CreateApiServerAndController(token string) error {
 			return nil
 		} else {
 			if tries < maxTries-1 {
-				glog.Info("Error scheduling machine controller. Will retry...\n")
+				glog.Infof("Error scheduling machine controller. Will retry... %v\n", err)
```

Comment:
This seems difficult to read. Perhaps something more like.
Retrying scheduling machine controller after getting error : %v

Reply:
Agree the message can be improved. I mainly just wanted to add the actual error because it was helpful tracking down some issues that were uncovered by this PR.

I will change the message. To something like this:

```
Retrying scheduling machine controller after encountering error: %v
```